webapp/src/components/conference/conference.tsx

```diff
@@ -0,0 +1,323 @@
+import * as React from 'react';
+
+import {Post} from 'mattermost-redux/types/posts';
+
+const BORDER_SIZE = 40;
+const POSITION_TOP = 'top';
+const POSITION_BOTTOM = 'bottom';
+const MINIMIZED_WIDTH = 320;
+const MINIMIZED_HEIGHT = 240;
+
+type Props = {
+    post: Post | null,
```

Comment:
Not really, because I want to define two different states, one is "post is null" so, there is no conference post open in the embedded environment, and the other is "post is not null" so, there is a conference post open in the embedded environment. If I want to "close" the embedded window, i set it to null, because setting something to "undefined" makes no sense. And actually, the reducer starts the state with `null` value, not with an undefined one.